ipynb/energy/EnergyModel_SystemEnergy.ipynb

```diff
@@ -79,11 +71,12 @@
     "import re\n",
     "import trappy\n",
     "\n",
-    "from collections import namedtuple, OrderedDict\n",
```

Comment:
I think the answer would be "no real reason in particular". I wanted the `clusters` dict to be populated automatically instead of manually, so I sketched a quick piece of code without minding the previous `namedtuple`. I could change that back to `namedtuple`, but I don't find the dict accessors to be that much of a nuisance (personal opinion here).

Reply:
Alright, I was just wondering. Automatic filling seems a good reason to switch :)